src/components/evaluationForm/index.tsx

```diff
@@ -0,0 +1,641 @@
+  const postData = async (data: IEvaluationData) => {
+    const formattedData = {
+      ...data,
+      prancha: minSecToPT(data.prancha),
+      forcaIsometricaMaos: minSecToPT(data.forcaIsometricaMaos),
+    };
+    const response = await post(
+      "avaliacao",
+      evaluationToApiPost(formattedData)
+    );
+    if (response?.status !== 201) {
+      throw new Error("Erro ao cadastrar avaliação");
+    }
+    //console.log(formattedData);
+    console.log("Avaliação cadastrada com sucesso");
+    reset();
+    reset({
+      prancha: "",
+      forcaIsometricaMaos: "",
+    });
+    console.log(
+      "Form POST submitted with data:",
+      evaluationToApiPost(formattedData)
+    );
+  };
+
+  const router = useRouter();
+
+  const onSubmit: SubmitHandler<IEvaluationData> = async (
+    data: IEvaluationData
+  ) => {
+    switch (method) {
+      case "PUT":
+        // console.log("clicado Alterar");
+        await updateData(data);
+        router.push("/relatorioAvaliacao");
+
+        break;
+      case "POST":
+        // console.log("clicado Cadastrar");
+        await postData(data);
+        break;
+    }
+  };
+
+  return (
+    <div className="flex items-center justify-center gap-2">
+      <form className="exercise-details" onSubmit={handleSubmit(onSubmit)}>
+        {/* Data */}
+        <div className="flex items-center justify-center gap-2 pb-5">
+          <label
+            htmlFor="nome"
+            className="inline-block w-14 text-center text-base font-semibold"
+          >
+            Data:
+          </label>
+          {method === "PUT" ? (
+            <input
+              type="date"
+              id="data"
+              className="w-32 flex rounded text-center"
+              value={date}
+              readOnly
+            />
+          ) : (
+            <input
+              type="date"
+              id="data"
+              className="w-36 flex rounded text-center"
+              {...register("data")}
+            />
+          )}
+        </div>
+        {errors.data && (
+          <span className="text-red-500 ml-2 flex items-center justify-left">
+            {errors.data.message}
+          </span>
+        )}
+        {/* Teste de Lunge */}
+        <div className="flex flex-col gap-2 items-end">
+          <div className="flex gap-8 justify-center">
+            <span className="text-right w-16 font-semibold text-xs pt-1 pl-5">
+              esquerdo
+            </span>
+            <span className="text-left w-16 font-semibold text-xs pt-1 pl-1">
+              direito
+            </span>
+          </div>
+          <div className="flex items-center ">
+            <label
+              htmlFor=""
+              className="inline-block w-48 text-left text-base font-semibold pr-1"
+            >
+              Teste de Lunge (Joelho):
+            </label>
+            {/* Teste de Lunge Esquerdo */}
+            <div className="flex w-36 gap-2">
+              <input
```

Comment:
Sempre que for um input que somente irá aceitar números, precisamos nos atentar para os dispositivo mobile.
O ideal é que o celular exiba somente aquele teclado número, sem exibir teclas com letras que não podem ser inseridas.
Para que isso aconteça, precisamos adicionas as seguintes propriedades HTML a tag <input>
![image](https://github.com/user-attachments/assets/b9a3ce95-e22b-4003-8e90-b207ec14e60a)


Podemos setar também o `type="number"` mas isso precisa ser bem analisado, uma vez que esse type costuma inserir um step button, e muitas vezes não é compatível com algumas bibliotecas como por exemplo o iMask.